k8s/upgrade/config/unsupported_versions.yaml

```diff
@@ -0,0 +1,3 @@
+unsupported_versions:
+  # add the list of unsupported versions as shown below
+  - 0.0.0
```

Comment:
Should we add v1 for completeness?
Do we support "wild cards" here btw?

Reply:
no wild cards support , the version in chart.yal is 0.0.0 so v1 not added here

Reply:
might be worth adding that, even as a follow-up PR